javaagent-api/src/main/java/io/opentelemetry/javaagent/instrumentation/api/db/cassandra/CassandraQueryNormalizer.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.api.db.cassandra;
+
+import io.opentelemetry.javaagent.instrumentation.api.db.sql.normalizer.ParseException;
+import io.opentelemetry.javaagent.instrumentation.api.db.sql.normalizer.SqlNormalizer;
```

Comment:
I don't like this new package structure. How many classes in total are there under `io.opentelemetry.javaagent.instrumentation.api.db` package? Do we really need to split them even more into several subpackages?

Reply:
Honestly I don't really like it as well, it's too long and there's too much nesting. How about `...api.db.normalizer`? The auto-generated code probably needs its own package, because that package needs to be excluded from spotbugs/checkstyle.

Reply:
Makes sense

Reply:
Done